### PR TITLE
refs[TASK][31151]: Create UI for Statistic Screen

### PR DESCRIPTION
**Pull request by Domhoan**
## Related Tickets
- [#Task 31151: Create UI for Statistic Screen](https://edu-redmine.sun-asterisk.vn/issues/31151)
## WHAT
- Create UI for Statistic Screen
## HOW
- Create UI for Statistic Screen: ProgresBar, RecyclerView
image

## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/tree/master/eng/android | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/res/layout/fragment_statistic.xml [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <View
+            android:id="@+id/view"
+            android:layout_width="0dp"
+            android:layout_height="@dimen/dp_48"
+            android:background="@android:color/white"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintHorizontal_bias="0.0"
```
Comment (NguyenTaTrung) [inline app/src/main/res/layout/fragment_statistic.xml]: thừa bias nhé

File: app/src/main/res/layout/fragment_statistic.xml [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/textTitleStatistic" />
+
+        <TextView
+            android:id="@+id/textHomeTeamStatistic"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="8dp"
+            android:layout_marginEnd="8dp"
```
Comment (NguyenTaTrung) [inline app/src/main/res/layout/fragment_statistic.xml]: hard code

File: app/src/main/res/layout/fragment_statistic.xml [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+            app:layout_constraintEnd_toStartOf="@+id/imageHomeTeamStatistic"
+            app:layout_constraintStart_toStartOf="@+id/view"
+            app:layout_constraintTop_toBottomOf="@+id/textTitleStatistic" />
+
+        <TextView
+            android:id="@+id/textAwayTeamStatistic"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="8dp"
+            android:layout_marginEnd="8dp"
```
Comment (NguyenTaTrung) [inline app/src/main/res/layout/fragment_statistic.xml]: hard code

File: app/src/main/res/layout/fragment_statistic.xml [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+            app:layout_constraintBottom_toBottomOf="@+id/imageAwayTeamStatistic"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toEndOf="@+id/imageAwayTeamStatistic"
+            app:layout_constraintTop_toTopOf="@+id/imageAwayTeamStatistic" />
+
+        <ImageView
+            android:id="@+id/imageHomeTeamStatistic"
+            android:layout_width="@dimen/dp_24"
+            android:layout_height="@dimen/dp_24"
+            android:layout_marginEnd="32dp"
```
Comment (NguyenTaTrung) [inline app/src/main/res/layout/fragment_statistic.xml]: h vẫn mắc lỗi này nhỉ

File: app/src/main/res/layout/fragment_statistic.xml [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+            app:layout_constraintBottom_toBottomOf="@+id/textHomeTeamStatistic"
+            app:layout_constraintEnd_toStartOf="@+id/guidelineCenterStatistic"
+            app:layout_constraintTop_toTopOf="@+id/textHomeTeamStatistic" />
+
+        <ImageView
+            android:id="@+id/imageAwayTeamStatistic"
+            android:layout_width="@dimen/dp_24"
+            android:layout_height="@dimen/dp_24"
+            android:layout_marginStart="32dp"
```
Comment (NguyenTaTrung) [inline app/src/main/res/layout/fragment_statistic.xml]: .

File: app/src/main/res/layout/fragment_statistic.xml [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+            android:id="@+id/recyclerStatistic"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:background="@android:color/white"
+            android:orientation="vertical"
+            android:overScrollMode="never"
+            app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintHorizontal_bias="0.0"
```
Comment (NguyenTaTrung) [inline app/src/main/res/layout/fragment_statistic.xml]: bỏ bias

File: app/src/main/res/layout/fragment_statistic.xml (hunk 4 above)
Comment (Domhoan) [reply app/src/main/res/layout/fragment_statistic.xml]: rõ ràng check hết rồi mà nhỉ, sao vẫn lòi ra mấy cái hard code này

**Pull request merged**